ros_ign_bridge/ros_ign_bridge/mappings.py

```diff
@@ -74,6 +74,9 @@
         Mapping('UInt32', 'UInt32'),
         Mapping('String', 'StringMsg'),
     ],
+    'rcl_interfaces': [
```

Comment:
It looks like this list is alphabetized, mind moving this up?

Reply:
Sure thing, I still don't know my alphabet :upside_down_face: